build script to build and test for package azure-pipeline-go

vikasgupta8 opened the pull request:
build script to build and test for package azure-pipeline-go

seth-priya commented:
@vikasgupta8 I tried to execute this script in a clean UBI container and it fails with the following error

[root@b0af0ac2fe70 azure-pipeline-go]# # Ensure go.mod file exists
[root@b0af0ac2fe70 azure-pipeline-go]# go mod init $PACKAGE_PATH
go: /azure-pipeline-go/go.mod already exists

does it work for you?

vikasgupta8 commented:
Yes it has failed. Previously I was getting the code using 'go get' command, but it was failing with junk character in name.
I have updated the script to use 'git clone', hence it failed.
Now I have removed the 'go mod init' command.

vikasgupta8 commented on a/azure-pipeline-go/azure-pipeline-go_ubi.sh:
```diff
@@ -0,0 +1,61 @@
+PACKAGE_VERSION=${1:-v0.2.2}
+PACKAGE_URL=https://github.com/Azure/azure-pipeline-go
+
+yum install -y git wget tar gcc-c++
```
Yes it require gcc.

seth-priya replied:
ok

seth-priya commented:
@vikasgupta8 looks like the branch has conflicts, can you please check and update?

seth-priya reviewed (commented):
Hi @vikasgupta8 I am not sure if the conflicts are resolved correctly, for example don't know why azure-pipeline-go has been added as a "submodule", can you maybe raise a fresh PR for this one?
